db/migrate/20230205145109_create_waterbodies.rb

```diff
@@ -0,0 +1,12 @@
+class CreateWaterbodies < ActiveRecord::Migration[6.1]
+  def change
+    create_table :waterbodies do |t|
+      t.string :name, null: false
+      t.string :description, null: false
```

Comment:
@dbrgn @Makram95 Should the description be nullable? Or do we always want to have a description?

Reply:
From a parser point of view, I think I'd favor a non-nullable field (which might be empty) over a nullable field. Otherwise you have to handle two different "empty states".

Reply:
I left it `null: false` for now. If it causes any problems we can easily change it.